src/common/CommonUtils.ts

```diff
@@ -49,7 +68,16 @@ export class CommonUtils {
                             `Error executing command '${command}':`
                         );
                         CommonUtils.logger.error(`${error}`);
-                        reject(error);
+
+                        // also include stderr & stdout for more detailed error
+                        let msg = error.message;
+                        if (stderr && stderr.length > 0) {
+                            msg = `${msg}\n${stderr}`;
```

Comment:
You probably want to identify these in the message too, e.g. ``msg = `${msg}\nstderr:\n${stderr}`;``, etc. I'm afraid the error message could become something of a jumbled mess otherwise.

Reply:
Also, though, I'm not sure that this is set up the way we want it. It feels like we should maybe log the additional information, but simply return the original error as output. This is a pretty general command, and depending on how consumers use it, it could end up dumping a lot of information out to e.g. a console or other UI, in a way that consumers couldn't really control.

Reply:
Very good point. I totally didn't think about that. Ok will update the logic here to only return the `error` itself but additionally log the `error + stderr + stdout`